.stylelintrc.json

```diff
@@ -40,6 +40,25 @@
         "max-width",
         "min-width",
         "display",
+        "appearance",
+        "flex",
+        "order",
+        "box-ordinal-group",
+        "box-direction",
+        "box-align",
+        "box-orient",
+        "box-pack",
+        "box-flex",
+        "flex-direction",
+        "flex-flow",
+        "flex-grow",
+        "flex-wrap",
+        "flex-shrink",
+        "flex-basis",
+        "justify-content",
+        "align-content",
+        "align-self",
+        "align-items",
```

Comment:
We don't need all these listed in the stylelint.json, and in particular, they shouldn't come before box-sizing.  Please revert this change.